_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-midnight
```

Comment:
What's this file?

Reply:
Let me share in case you run into this again.  I was really stumped on this because I would remember creating a file and committing it, but I didn't.  Apparently while looking for Dark Mode on github.com, I ran into theme selector for "GitHub Pages".   There are icons of various themes, and a button that says "Select Theme."  This button actually creates the _config.yml file in your repo and commits it, without such indication.  Anyway I'll remove the file, but I thought it might save you some trouble in future.